crates/bevy_gltf/src/loader.rs

```diff
@@ -1415,12 +1418,16 @@ fn load_node(
 
 /// Returns the label for the `mesh`.
 fn mesh_label(mesh: &gltf::Mesh) -> String {
```

Comment:
Doesn't have to be done in this PR, but at this point we could just remove those functions and use the enums directly in their place.